update contributing text (#823)
feltjs/felt-server: contributing.md

# Contributing to `@feltjs/felt-server`

Thanks for your interest in helping!
To keep our development speed up,

> [@feltcoop](https://github.com/feltcoop) makes most decisions in private,

Reviewer: Should this be @feltjs instead, since that's technically the org responsible for the software project?

> but our goal is to have open community-driven processes.

Reviewer: ```suggestion
but we are planning to develop open community-driven processes soon.
```

> Our code is open source, so we publish it as we write it,
> but for now most decisions are made privately.

Reviewer: ```suggestion
Our code is open source, so we publish it as we write it.
```
Deleted the second line since it seemed redundant with the new "we make private decisions" line above.
